codejam2/TicTacToe.cs

```diff
@@ -58,11 +58,18 @@ public static void PlayGame()
             string figletText = FiggleFonts.Chiseled.Render("It's a draw!");
             Colorful.Console.WriteLine(figletText);
         }
+        string figletText = FiggleFonts.Chiseled.Render("Press r to restart!");
+        Colorful.Console.WriteLine(figletText);
+        char restart = char.Parse(Console.ReadLine());
+        if (restart == 'r' || restart =='R')
+        {
+            Program.Main();
+        }
```

Comment:
This restart functionality is good since it allows the user to play the game again without having to run the application again, and I like that both lowercase and uppercase r is accepted.